app/main.py

```diff
@@ -15,15 +15,6 @@
 
 from app.script_tracking import ScriptMaster
 
-import app.dist_lock as dist_lock
```

Comment:
Removing the dist.lock makes sense. It's not used efficiently throughout the code and we won't have millions of users.

Reply:
second, good job.  It looks simpler and clearer